src/v/compression/stream_zstd.cc

```diff
@@ -29,7 +29,8 @@ namespace compression {
     if (rc == ZSTD_error_memory_allocation) {
         ss::throw_with_backtrace<std::bad_alloc>();
     } else {
-        ss::throw_with_backtrace<std::runtime_error>(fmt::format("ZSTD error:{}", ZSTD_getErrorName(rc)));
```

Comment:
Did this line actually change or is it whitespace noise?

Reply:
No, I did not change anything